### PR TITLE
Add pygments package

### PR DESCRIPTION
**Pull request by msabramo**
Sample usage:

```js
languagePluginLoader.then(() => {
  // pyodide is now ready to use...
  pyodide.loadPackage('pygments').then(() => {
    pyodide.runPython(`
from pygments import highlight
from pygments.lexers import PythonLexer
from pygments.formatters import HtmlFormatter

code = 'print "Hello World"'
print(highlight(code, PythonLexer(), HtmlFormatter()))
    `);
  });
});
```

Screen Shot 2019-03-14 at 11 34 49 PM

### CODE REVIEW

**Comment by mdboom** [comment]
Thanks for the PR.  It looks like the failure is unrelated.  I'll have a fix for that shortly.

File: packages/pygments/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package:
+  name: pygments
```
Comment (mdboom) [inline packages/pygments/meta.yaml]:
```suggestion
  name: Pygments
```

This needs to match the tarball name.  Also, the `packages/pygments` directory needs to move to `packages/Pygments`.

Comment (msabramo) [reply packages/pygments/meta.yaml]: Updated. Thanks!

**Comment by msabramo** [comment]
Are these correctable in my PR or are they just intermittent failures?

Screen Shot 2019-03-15 at 11 04 57 AM

**Comment by mdboom** [comment]
I've never seen the firefox one fail like that before, but it seems unlikely to be related to the PR.  I just restarted the workflow on Circle-CI.

**Comment by msabramo** [comment]
Hmmm, now it's just `test-firefox` that failed. I guess that's an improvement, since before it was that one plus `benchmark` that failed.

Screen Shot 2019-03-15 at 12 01 13 PM

**Pull request merged**